src/utils/SortingNetworkGenerator.java

```diff
@@ -61,9 +66,12 @@ public static boolean verifyPythonVersionAndDialog() {
     }
 
     public static boolean encodeNetwork(int[] indices, String path) {
+		if (indices.length == 0 || indices.length > LIMIT) {
```

Comment:
Won't there be weird results if `indices.length` equals 1? (Yes, that should never happen, but you also modified lines 73-74)

Reply:
fixed